.github/workflows/deploy_latest_release.yml

```diff
@@ -58,3 +70,43 @@ jobs:
           PROJECT: pwa-kit
           MESSAGE: "tag ${{ github.event.release.tag_name }}"
           FLAGS: --cloud-origin https://cloud-testing.mobify-staging.com -c ~/.mobify --wait
+
+  deploy-demo-site:
+    needs: check-latest-release
+    runs-on: ubuntu-latest
+    if: needs.check-latest-release.outputs.IS_LATEST_RELEASE == 'true'
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Setup node
+        id: setup_node
+        uses: actions/setup-node@v4
+        with:
+          node-version: 22
+          cache: "npm"
+
+      - name: Install Monorepo Dependencies
+        run: |-
+          # Install node dependencies
+          node ./scripts/gtime.js monorepo_install npm ci
+
+      - name: Generate Retail App Demo
+        uses: ./.github/actions/e2e_generate_app
+        with:
+          PROJECT_KEY: "retail-react-app-demo-site"
+
+      - name: Create MRT credentials file
+        uses: "./.github/actions/create_mrt"
+        with:
+          mobify_user: ${{ secrets.MOBIFY_CLIENT_USER }}
+          mobify_api_key: ${{ secrets.MOBIFY_CLIENT_API_KEY }}
+
+      - name: Push Bundle to MRT (Demo Site)
+        uses: "./.github/actions/push_to_mrt"
+        with:
+          CWD: "../generated-projects/retail-react-app-demo-site"
+          TARGET: production
+          PROJECT: scaffold-pwa
+          MESSAGE: "tag ${{ github.event.release.tag_name }}"
```

Comment:
Can we add a more useful message to the bundle?
<img width="1125" alt="Screenshot_2025-03-11_at_4_19_48 PM" src="https://github.com/user-attachments/assets/c518ccb4-f66e-4b47-9b49-55626dcc03fa" />


Right now looks like the value for `github.event.release.tag_name` gets lots, so we end up just with "tag".

Reply:
This is a quirk resulting from triggering the build using git push to a branch rather than the intended release publish.

I'll add a fallback name if the release.tag_name is blank

Reply:
Fallback name has been added. The fallback is to use the build id, which is the same as what we use in other workflows.